.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
+      - uses: bnjbvr/cargo-machete@main
```

Comment:
We should put this below `actions/cache` for it to get cached eventually? Otherwise, it'd add 45s to every run :sweat_smile:.